cli/import-jdl.js

```diff
@@ -248,23 +252,14 @@ const shouldTriggerInstall = (generator, index) =>
 const isAllCompleted = items => Object.values(items).every(it => it);
 
 class JDLProcessor {
-    constructor(jdlFiles, options) {
-        logger.debug(`JDLProcessor started with jdlFiles: ${jdlFiles} and options: ${toString(options)}`);
+    constructor(jdlFiles, jdlContent, options) {
+        logger.debug(`JDLProcessor started with content: ${jdlFiles} and options: ${toString(options)}`);
```

Comment:
I don't think the log helps a lot in this case. Maybe telling that it started with files and/or content and options is a good idea. For instance: if I call the JDLProcessor with only files, the log tells me that the processor started with files and options (no content); or if only with a content that the processor started with only a content and options.

Reply:
Sure